src/index.js

```diff
@@ -0,0 +1,287 @@
+import './index.css';
+import './assets/img/logo-transparent.png';
+import { getData, getMovieData } from './modules/api.js';
+import {
+  getLikes, getComments, setComment, setLikes,
+} from './modules/involvement.api.js';
+
+const movieList = document.querySelector('.movie-list');
+const movieDetails = document.querySelector('.movie-details');
+const page = document.documentElement;
+const searchMovie = document.querySelectorAll('.search-input');
+const menuButton = document.querySelector('.mobile_menu_btn');
+const mobileMenu = document.querySelector('.mobile_menu');
+const remButton = document.querySelector('.menu_close');
+const prev = document.querySelector('.prev');
+const next = document.querySelector('.next');
+let movieName = 'marvel';
+let prevNum = 2;
+let nextNum = 2;
+
+// Mobile Menu
+// transition styles
+mobileMenu.style.right = '-800px';
+mobileMenu.style.transition = 'right 800ms';
+
+// listen for a click on the menu button and X button
+menuButton.addEventListener('click', () => {
+  mobileMenu.style.display = 'block';
+  setTimeout(() => {
+    mobileMenu.style.right = '0px';
+  }, 1);
+});
+
+remButton.addEventListener('click', () => {
+  mobileMenu.style.right = '-800px';
+  setTimeout(() => {
+    mobileMenu.style.display = 'none';
+  }, 900);
+  menuButton.style.display = 'block';
+});
+
+// Function to count number of element on the page
+const countItems = (items) => {
+  let movieCount = 0;
+  let serieCount = 0;
+  let gameCount = 0;
+  items.forEach((item) => {
+    if (item.Type === 'movie') {
+      movieCount += 1;
+    } else if (item.Type === 'series') {
+      serieCount += 1;
+    } else if (item.Type === 'game') {
+      gameCount += 1;
+    }
+  });
+  const moviesCount = document.querySelectorAll('.movies-count');
+  const seriesCount = document.querySelectorAll('.series-count');
+  const gamesCount = document.querySelectorAll('.games-count');
+  moviesCount.forEach((counter) => {
+    counter.innerHTML = movieCount;
+  });
+  seriesCount.forEach((counter) => {
+    counter.innerHTML = serieCount;
+  });
+  gamesCount.forEach((counter) => {
+    counter.innerHTML = gameCount;
+  });
+  return { movieCount, serieCount, gameCount };
+};
+
+// Function to get & display lakes on the home page
+
+const likeCount = (id, index, Likes) => {
+  let like = 0;
+  Likes.forEach((data) => {
+    if (data.item_id === id) {
+      like = data.likes;
+    }
+  });
+  document.querySelectorAll('.likes-data').forEach((card, i) => {
+    if (index === i) {
+      card.innerHTML = like;
+    }
+  });
+};
+
+const commentsCount = (comments) => {
+  let count = 0;
+  if (comments.length > 0) {
+    comments.forEach(() => {
+      count += 1;
+    });
+  } else {
+    count = 0;
+  }
+  return count;
+};
+
+const showComments = async (id) => {
```

Comment:
- [ ] Please update  on the screen after submitting them, for now it is now updating the screen unless you close it and the reopen it

Reply:
We will be working on this very soon thanks :pray: